travis/push-to-cicd-status.sh

```diff
@@ -97,9 +97,18 @@ git_clone_repo
 
 while true; do
     if [[ ${TRAVIS_REPO_SLUG##*/} != "acc-provision" ]]; then
-        add_artifacts
-        update_container_release
-        add_trivy_vulnerabilites
+        if ! add_artifacts; then
```

Comment:
These function calls will only return non-zero for either a command that is error checked or the last commands executed in the functions.
For `add_artifacts` and `add_acc_provision_artifacts` functions we can get non zero status from last commands only, should we `return 1` from these functions for the commands we expect to fail and return non-zero like git pull. curl for logs.

Reply:
this is an implicit way, so any commands in the above function fail results in if <check> true, default will be false. The same is done  inline 121, 125 which is being tested out